### PR TITLE
Start hook explanation

### PR DESCRIPTION
**Pull request by lpatiny**

### CODE REVIEW

File: docs/hooks/intro.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+          ]}
+        />
+        <Annotations>{zoom.annotations}</Annotations>
+      </Plot>
+    );
+  }
+}
+```
+
+```jsx live
```
Comment (lpatiny) [inline docs/hooks/intro.md:35]: I'm trying to have a simple example in which we can display the tracking events in a DIV.

However when I use the following code as soon as I mouse down there is a refresh of the component and therefore the mouse move is lost.

What is the simple way to solve this problem ?

Comment (stropitek) [reply docs/hooks/intro.md:35]: Where is this page supposed to be? I cannot find it in the deploy preview

File: docs/hooks/intro.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    <PlotController>
+      <Tracking />
+      <EventInfo />
+    </PlotController>
+  );
+
+  function EventInfo() {
+    return (
+      <>
+        {currentEvent && (
```
Comment (targos) [inline docs/hooks/intro.md:49]: You should generally not declare components inside components (here `EventInfo` and `Tracking` are nested in `Demo`). This results in the components being recreated every time Demo is rendered (when the state changes). When a new component replaces another one, React will completely unmount the DOM, which triggers cleanup effects, resets the internal states, and may also impact the events emitted by the browser

Comment (lpatiny) [reply docs/hooks/intro.md]: Thanks. I found finally the correct way (I think) to do it with docusaurus and react-live.

**Pull request merged**